ui/packages/catalog-ui-search/src/main/webapp/react-component/location/location.js

```diff
@@ -88,6 +120,16 @@ const LocationInput = props => {
       </DropdownPadding>
       <Form>
         {Input !== null ? <Input {...props} /> : null}
+        {errors ? (
+          <Invalid>
+            &nbsp;
+            <span className="fa fa-exclamation-triangle" />
+            &nbsp; {inValidInput} is not an acceptable {inValidKey} value.
```

Comment:
It appears that `inValidKey` is not appearing in the error text.
![image](https://user-images.githubusercontent.com/4467636/57395927-17c33d80-717e-11e9-8116-ba42a978c275.png)

Reply:
Weird it was working for me as you can see in the screenshot on my PR, but I wonder if there's some funky behavior for DMS coordinates as opposed to DD coordinates. I'll check it out